docs/deposit_and_withdrawal.md

```diff
@@ -0,0 +1,102 @@
+# Deposit and withdrawal
+
+Deposit and withdrawal is a special layer1 <-> layer2 messaging mechanism with assets transfer. Users can deposit assets from layer1 (CKB) to layer2 (godwoken), or withdraw from layer2 back to layer1.
+
+## Deposit
+
```

Comment:
> A deposit request is a layer1 cell but not a transaction

How about “A deposit request is created by sending a layer1 transaction which ...”

Reply:
Update the description.

I think we should use 'deposit lock' rather than 'deposit-lock', lock is a concept of CKB, the deposit lock implies the script is the `lock` field of a CKB cell. But the 'deposit-lock' implies it is a unique concept no related to the CKB's lock, which isn't true.

Reply:
Both are okay to me